### DISCUSSION TITLE
Two new buttons?

### DISCUSSION THREAD

**Discussion by mauriva** [TT-ReBORN/Georgia-ReBORN#104] [Ideas]
I am testing the new version v2.3.0 Beta 11b.
I found no problems at the moment, just one request.
Compared to other skins I am missing these two buttons: 
1) the one for adding more music to the playlist (from files, from folders, ...);
2) a button for the 'Clear' command.

As this is actions I perform most often in foobar, having the commands 'at hand' would be really nice.
Thanks for the great work!

**Comment by TT-ReBORN** [comment]
Hi [mauriva](https://github.com/mauriva),

I don't really understand these request, because adding more music to the playlist can be done: 
1. Through drag and drop files or folders via your Window Explorer to the active playlist
2. Through top menu File > Add Files / Add Folders
3. Through your the Library panel > Double click on album or song or right click on for context menu > Add to current playlist/Add to new playlist

To remove playlist items:
1. Album - click on thumbnail to select all songs from album right click for context menu > Remove
2. Song -  click on individual song right click for context menu > Remove
3. Everything - select something from active playlist and hit CTRL + A to select everything and right click for context menu > Remove

-TT

**Reply by mauriva** [reply]
Thank you for answer.
For the first point (add music) I use always the top menu "File > Add Files / Add Folders".
For the second point I use always the top menu "Edit > Clear".
What I'm asking is for two buttons that act as "shortcuts" for these two commands.

**Reply by TT-ReBORN** [reply]
That is absolutely not necessary because the theme has already too much buttons and no space. 
You can setup these and other shortcuts via "Keyboard Shortcuts" in foobar's Preferences > Keyboard Shortcuts.
